shared/src/main/scala/scorex/crypto/authds/avltree/batch/PersistentBatchAVLProver.scala

```diff
@@ -48,7 +48,7 @@ object PersistentBatchAVLProver {
                    ): Try[PersistentBatchAVLProver[D, HF]] = Try {
 
     new PersistentBatchAVLProver[D, HF] {
-      override var avlProver: BatchAVLProver[D, HF] = avlBatchProver
+      var avlProver: BatchAVLProver[D, HF] = avlBatchProver
```

Comment:
Why "override" is removed? It is better to be explicit about overriding.

Reply:
Well, this is why overrides should be explicit in the code.
I suggest to change the base trait declaration to follow this pattern:

```scala
trait Ok2 {
  def i: Int
  def i_=(v: Int): Unit
}
class Ok2C extends Ok2 { override var i: Int = 1 }
```